### ISSUE TITLE
terminate called after throwing an instance of 'SmartRedis::RuntimeException'

### ISSUE THREAD

**Issue by syedalihasany** [CrayLabs/SmartRedis#418]
# Description
C++ Code to set model from file:
client.set_model_from_file(model_key, model_file, "TORCH", "CPU", 1000);
__________________________
it throws a runtime error
terminate called after throwing an instance of 'SmartRedis::RuntimeException'
what():  Redis error when executing command: ERR unknown command `AI.CONFIG`, with args beginning with: `GET`, `MODEL_CHUNK_SIZE`, 
Aborted (core dumped)

# System
 - OS: Ubuntu
 - OS version: 20.04
 - Commit/version of library:

**Comment by billschereriii** [comment]
Hi, thanks for reaching out!

We recently upgraded our RedisAI requirement to version 1.2.7 in order to capture the ability to view the current model chunking setting. The message you're showing here looks like you're using an older version of RedisAI that doesn't support the `AI.CONFIG GET MODEL_CHUNK_SIZE` command (which would be versions 1.2.5 and earlier, if memory serves correctly).

If you can upgrade to RedisAI version 1.2.7, that should solve the problem. The easiest way to do that would be to do a full rebuild of SmartRedis after doing a `make clobber` -- the Makefile will then download the updated version for you automatically.

Please let me know if this doesn't help or if you can still reproduce the problem after the update.

**Comment by syedalihasany** [comment]
Hi Bill,

Thanks for getting back to me. I think that issue was resolved after I
cleaned and rebuilt Smartsim and started the orchestrator through the
python interpreter. I have however run into a new issue that I have posted
in the issues for Smartsim but Al Rigazzi from your team contacted me about
it and I am yet to make the changes. I'll keep you updated in case I can
not get it to work.

However, I would like to appreciate that you guys respond so quickly with
meaningful feedback. Really appreciate it.


Yours sincerely,
Ali
